lib/vdsm/storage/lvm.py

```diff
@@ -1440,6 +1440,62 @@ def getAllVGs():
     return _lvminfo.getAllVgs()  # returns list
 
 
+def check_single_vg_autoactivation(vg_name):
+    """
+    Retrieve the autoactivation field of vg_name and check if is disabled.
+    For each VG with autoactivation enabled, log a warning and disable it.
+    The autoactivation field should remain always disabled to avoid unexpected
+    VG activation.
+
+    Args:
+        vg_name (str): name of the VG to check.
+
+    Raises:
+        se.VolumeGroupDoesNotExist: in case it fails at retrieving the
+            autoactivation field of a VG.
+        se.VolumeGroupDisableOptionError: in case it fails at disabling the
+            autoactivation field of a VG.
+    """
+    cmd = ["vgs", "-o", "autoactivation", vg_name]
+    try:
+        out = _lvminfo.run_command(
+            cmd, devices=_lvminfo._getVGDevs([vg_name]))
+    except se.LVMCommandError as e:
+        log.warning(
+            "Retrieving VG %s autoactivation field failed", vg_name)
```

Comment:
Maybe make it `'%s'` or `%r` to make the name easier found in the log message? Nir?
If so, relevant for other logs as well.